content/en/docs/configuration/externalloadbalancer.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "External Load Balancer"
+linkTitle: "External Load Balancer"
+weight: 40
+type: "docs"
+---
+
+When you are using an external load balancer provided by any host, you can face several configuration issues to get it work with Cert Manager.
+
+This documentation is meant to help in configure HTTP-01 type of challenges for instances behind external load balancer.
+
+## NAT Loopback / Hairpin
+
+The first configuration point is NAT loopback. You can face `pre check` issues due to Load Balancer preventing instances behind him to access his external interface.
+
+Some Network Load Balancer have this kind of limitation for several reasons. It can be configured through iptables rerouting configuration known as `NAT loopback`.
+
+To check if you are facing this problem :
+
+1. Check that the challenge endpoint is publically accessible : `curl <endpoint>`
+2. Check that the challenge endpoint is NOT accessible from inside behind the Load Balancer : Use SSH to open a session on a node places behind the LB ; then launch the same command than before : `curl <endpoint`
+
+The HTTP-01 challenge endpoint can be found in the logs when the `pre-check` fails. If it does not appear in the logs, you can check the challenge URL by `kubectl`command.
```

Comment:
I would not recommend looking in the logs for this at all.
I would replace this part by looking at either the challenge or ingress resourve